lib/matplotlib/widgets.py

```diff
@@ -1585,8 +1589,11 @@ def __init__(self, ax, labels, active=0, activecolor=None, *,
 
             .. versionadded:: 3.7
 
-        label_props : dict or list of dict, optional
-            Dictionary of `.Text` properties to be used for the labels.
+        label_props : dict of lists, or list of dict, optional
```

Comment:
I tried passing a list of dict, but that throws the error `TypeError: 'label_props' must be an instance of dict or None, not a list`. So I think this has to be restricted to just a dict of lists, and *not* a list of dict?